cpp/tests/scalar/scalar_device_view_test.cu

```diff
@@ -47,6 +47,7 @@ __global__ void test_value(ScalarDeviceViewType s,
   *result = (s.value() == s1.value());
 }
 
+#if 0
```

Comment:
Had to skip some of the scalar_device_view tests which assumed the view was mutable.  The skipped tests could be reworked to make them relevant.